<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExportDialog</class>
 <widget class="QWidget" name="ExportDialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>264</width>
    <height>434</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OpenSprite Export</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="1" column="0" colspan="3">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Attribute Format</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QRadioButton" name="radio_spritepad">
          <property name="text">
           <string>SpritePad</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QRadioButton" name="radio_empty">
          <property name="text">
           <string>Empty</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QRadioButton" name="radio_seuck">
          <property name="text">
           <string>SEUCK</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>File Format</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout">
        <item row="1" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Start Address:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QSpinBox" name="address">
          <property name="minimum">
           <number>0</number>
          </property>
          <property name="maximum">
           <number>65535</number>
          </property>
          <property name="singleStep">
           <number>64</number>
          </property>
          <property name="value">
           <number>12288</number>
          </property>
          <property name="displayIntegerBase">
           <number>16</number>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="3">
         <widget class="QRadioButton" name="radio_prg">
          <property name="text">
           <string>PRG Format</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="4" column="0" colspan="3">
         <widget class="QRadioButton" name="radio_bin">
          <property name="text">
           <string>BIN Format</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>$</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="button_export">
     <property name="text">
      <string>Export</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QPushButton" name="button_close">
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>radio_prg</sender>
   <signal>toggled(bool)</signal>
   <receiver>address</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>66</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>185</x>
     <y>59</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
